nucleus/modelci/__init__.py

```diff
@@ -1,4 +1,19 @@
 """Model CI Python Library."""
+
+__all__ = [
+    "AvailableEvalFunctions",
+    "Connection",
+    "EvalFunctionEntry",
+    "EvaluationCriterion",
+    "GetEvalFunctions",
+    "ModelCI",
+    "ThresholdComparison",
+    "UnitTest",
+    "UnitTestEvaluation",
+    "UnitTestItemEvaluation",
+    "UnitTestMetric",
```

Comment:
imo we can just have `ModelCI` and `UnitTest` here for now. The others are rather confusing for the beginning. Wdyt @sasha-scale ?